### ISSUE TITLE
Refresh timeout not working

### ISSUE THREAD

**Issue by Rastutatis** [dubravcik/pbixrefresher-python#20]
When adding a Refresh-timeout parameter, the default one is still being used, causing a timeout in big reports.

**Comment by dubravcik** [comment]
Yep, there was a bug, working now for me. Make sure to type `--refresh-timeout` as the parameter.
Thanks
